scripts/properties.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python3
```

Comment:
How do we prevent users from forgetting to call properties.py to update this file after adding a style?

Reply:
I'm planning to add property.py to CI similar to lv_conf_generate.py. Let's firstly review if the proposed method is the right way.